src/main/grammars/RustParser.bnf

```diff
@@ -604,7 +604,7 @@ PatField ::= PatFieldFull | box? PatBinding
 
 BindingMode ::= ref mut? | mut
 
-upper Constant ::= (static mut? | const) (identifier | '_') TypeAscription [ '=' AnyExpr ] ';' {
+upper Constant ::= (static mut? | const) (identifier | '_') TypeAscription? [ '=' AnyExpr ] ';' {
```

Comment:
Don't forget to increment `org.rust.lang.core.parser.RustParserDefinition#PARSER_VERSION` to update all nesseary indixes after plugin update